spec/ddtrace/contrib/active_storage/tracer_spec.rb

```diff
@@ -0,0 +1,195 @@
+      self.abstract_class = true
+    end)
+
+    stub_const('Article', Class.new(ApplicationRecord) do
+      # active storage initializers have changed from 5 => 6 so we need to mimic both approaches
+      if Rails.version >= '6.0'
+        # https://github.com/rails/rails/blob/901f12212c488f6edfcf6f8ad3230bce6b3d5792/activestorage/lib/active_storage/engine.rb#L87
+        include ActiveStorage::Attached::Model
+        include ActiveStorage::Reflection::ActiveRecordExtensions
+        # https://github.com/rails/rails/blob/901f12212c488f6edfcf6f8ad3230bce6b3d5792/activestorage/lib/active_storage/engine.rb#L142
+        ActiveRecord::Reflection.singleton_class.prepend(ActiveStorage::Reflection::ReflectionExtension)
+      else
+        # https://github.com/rails/rails/blob/2a3f8a29e5d7d17a0448fa47fd756c8d11b175b1/activestorage/lib/active_storage/engine.rb#L71
+        extend ActiveStorage::Attached::Macros
+      end
+
+      has_one_attached :image
+    end)
+
+    # This is needed to ensure tests write to disk and not make http requests
+    Rails.configuration.active_storage.service_configurations = {
+      local: {
+        service: 'Disk',
+        root: '/tmp/dd-trace-rb/storage'
+      }
+    }
+
+    ActiveRecord::Base.establish_connection(mysql_connection_string)
+
+    begin
+      Article.count
+    rescue ActiveRecord::StatementInvalid
+      ActiveRecord::Schema.define(version: 20180101000000) do
+        create_table 'articles', force: :cascade do |t|
+          t.string   'title'
+          t.datetime 'created_at', null: false
+          t.datetime 'updated_at', null: false
+        end
+
+        create_table :active_storage_blobs do |t|
+          t.string   :key,        null: false
+          t.string   :filename,   null: false
+          t.string   :content_type
+          t.text     :metadata
+          t.bigint   :byte_size,  null: false
+          t.string   :checksum,   null: false
+          t.datetime :created_at, null: false
+
+          t.index [:key], unique: true
+        end
+
+        create_table :active_storage_attachments do |t|
+          t.string     :name,     null: false
+          t.references :record,   null: false, polymorphic: true, index: false
+          t.references :blob,     null: false
+
+          t.datetime :created_at, null: false
+
+          t.index [:record_type, :record_id, :name, :blob_id],
+                  name: 'index_active_storage_attachments_uniqueness',
+                  unique: true
+        end
+      end
+    end
+
+    ApplicationRecord
+  end
+
+  before { app }
+
+  let(:configuration_options) { {} }
+
+  before(:each) do
+    # Prevent extra spans during tests
+    # This is needed to ensure tests write to disk and not make http requests
+    # Rails.configuration.active_storage.service_configurations = {
+    #   local: {
+    #     service: 'Disk',
+    #     root: '/tmp/dd-trace-rb/storage'
+    #   }
+    # }
```

Comment:
Is this block supposed to be commented out?

Reply:
err, i think it should've been deleted, will update